### ISSUE TITLE
Hero Point Handler stopped working

### ISSUE THREAD

**Issue by Wobbelblob** [xdy/xdy-pf2e-workbench#1254]
Few weeks ago, the Hero Point Handler simply stopped working. I get the confirmation that it did what I wanted to but the changes does not go through to the player sheets. Even with only the Workbench enabled nothing changes.

Foundry Version: Version 11 Stable 315
Pathfinder Version: 5.13.6
Workbench Version: 5.56.2

I do have to mention that I host the game over Forge. That includes one permanently on module. But until now it never made any problems, so I am not sure if it is relevant.

**Comment by xdy** [comment]
I can't reproduce that using the same versions. I tried resetting all, adding to all, adding to one player, all worked for me.
Do you get any errors (red text) in the console?

**Comment by Wobbelblob** [comment]
No, no errors or even warnings. Interestingly it still doesn't work on a seperate Foundry installation, the one I host from my desktop and had a really old version of the campaign.

Maybe something got disconnected somewhere, because I just saw that section 2 of the HPH has "no characters", even with more than 4 being present in the regular character list. 

Edit: I think I found the issue. At some point, the system added a red folder called "The Party" that has the general party sheet or something like that? My regular campaign does not have that folder and I cannot recreate it. If I put the character sheets into that folder, it works. Any idea how I can make that folder? Seems to not be a normal folder.
